theme_vehicle/views/customizations.xml

```diff
@@ -1,171 +1,365 @@
+            <attribute name="data-shape-colors">o-color-1;;;;o-color-5</attribute>
+        </xpath>
+    
+        <xpath expr="//div[hasclass('row')]/div[2]" position="attributes">
+            <attribute name="class" add="col-lg-5 offset-lg-1" remove="col-lg-6" separator=" "/>
+        </xpath>
+        
+        <xpath expr="//h2" position="before">
+            <h4 class="text-o-color-1"><b>The world is yours</b></h4>
+        </xpath>
+
+        <xpath expr="//h2" position="replace" mode="inner">
+            No compromise
+        </xpath>
+
+        <xpath expr="//p" position="replace" mode="inner">
+            Performance and design were the key words during the conception of this super car. No compromises were made to keep the pleasure of driving despite those constraints.
+        </xpath>
+        <xpath expr="//p[2]" position="replace" mode="inner">
+            Take advantage of a free track trial to discover it.
+        </xpath>
+    </template>
+
+    <!-- ======== IMAGE - TEXT ======== -->
+    <template id="s_image_text" inherit_id="website.s_image_text" name="Vehicle s_image_text">
+        <xpath expr="//section" position="attributes">
+            <attribute name="class" add="pb56 pt24 o_cc o_cc1" remove="pt32 pb32" separator=" "/>
+        </xpath>
+
+        <xpath expr="//img" position="attributes">
+            <attribute name="src">/web_editor/image_shape/website.s_image_text_default_image/web_editor/float/flt_square_2.svg?c2=o-color-1&amp;c5=o-color-5</attribute>
+            <attribute name="data-shape">web_editor/float/flt_square_3</attribute>
+            <attribute name="data-original-mimetype">image/jpeg</attribute>
+            <attribute name="data-file-name">s_image_text.svg</attribute>
+            <attribute name="data-shape-colors">;o-color-1;;;o-color-5</attribute>
+        </xpath>
+
+        <xpath expr="//div[hasclass('row')]/div[2]" position="attributes">
+            <attribute name="class" add="col-lg-5 offset-lg-1" remove="col-lg-6" separator=" "/>
+        </xpath>
+
+        <xpath expr="//h2" position="before">
+            <h4 class="text-o-color-1"><b>A greener lifestyle</b></h4>
+        </xpath>
+
+        <xpath expr="//h2" position="replace" mode="inner">
+            Electric Driving
+        </xpath>
+
+    </template>
+
+    <!-- ======== NUMBERS ======== -->
+    <template id="s_numbers" inherit_id="website.s_numbers">
+        <!-- Section -->
+        <xpath expr="//section" position="attributes">
+            <attribute name="class" add="o_cc4 pt48 pb48" remove="o_cc2 pt24 pb24" separator=" "/>
+            <attribute name="data-oe-shape-data">{"shape":"web_editor/Rainy/09_001","flip":[]}</attribute>
+        </xpath>
+        <!-- Shape -->
+        <xpath expr="//div[hasclass('container')]" position="before">
+            <div class="o_we_shape o_web_editor_Rainy_09_001"/>
+        </xpath>
+    </template>
+
+
```

Comment:
Sometimes you put two blank lines, sometimes one between template -> only one always :+1: